hawkular-alerts-rest-tests/pom.xml

```diff
@@ -219,6 +226,7 @@
               <excludes>
                 <excludeFile>org/hawkular/alerts/rest/ClusterITest.class</excludeFile>
                 <excludeFile>org/hawkular/alerts/rest/PerfCrudITest.class</excludeFile>
+                <excludeFile>org/hawkular/alerts/rest/CORSITest.class</excludeFile>
```

Comment:
Minor.
Is this necessary ? I saw below that test is introduced on the IntegrationSuite.